card/spec/lib/card/set/trait_spec.rb

```diff
@@ -0,0 +1,62 @@
+
+describe Card::Set::Trait do
+  class Card
+    module Set
+      module Type
+        module Phrase
+          extend Card::Set
+          card_accessor :write, type: :pointer
+          card_accessor :read, type_id: Card::PointerID
```

Comment:
dumb question #2.  This is not supposed to be card_reader and card_writer, right?  You actually want a card name named +*read and/or +*write?

dumb question #3.  Does the `*write` card exist?  Or another card with the codename "write"?  `*read` exists for permission rules, of course.  If it doesn't exist, Wagn should probably fail loudly...